jsonpb/jsonpb_test.go

```diff
@@ -264,6 +264,18 @@ var (
 		`}`
 )
 
+// Bytes is used to test that []byte type aliases are serialized to base64.
+type Bytes []byte
+
+// BytesMessage is used to test that []byte type aliases are serialized to base64.
+type BytesMessage struct {
```

Comment:
Aha yes it would be better if this message was generated by gogoprotobuf.  Thank you @tamird